jabgui/src/main/java/org/jabref/gui/documentviewer/DocumentViewerViewModel.java

```diff
@@ -89,11 +108,34 @@ private void setCurrentDocument(Path path) {
         }
     }
 
+    private boolean isPdfFile(LinkedFile file) {
+        if (file == null || file.getLink() == null || file.getLink().trim().isEmpty()) {
+            return false;
+        }
+
+        try {
+            Path filePath = Path.of(file.getLink());
+            return FileUtil.isPDFFile(filePath);
+        } catch (Exception e) {
```

Comment:
do not catch generic exceptions, be very specific. You don't know then why sthg failed? 1000% sure it is because it is not a pdf file?